doc/Usage.md

```diff
@@ -0,0 +1,75 @@
+# Using Custom Student Verification App
+
+## Theme Customization
```

Comment:
@shimulch, since this section is primarily aimed at the developers, can we move this to the last part of the document and keep the actual usage instructions at the top?

Reply:
Moved to the end of the document.